packs/equipment/emerald-fulcrum-lens.json

```diff
@@ -34,7 +34,23 @@
             "title": "Pathfinder #165: Eyes of Empty Death"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
```

Comment:
Should this be on an effect instead, since it's while you have it invested?

Reply:
Yes, the lenses are weird in that you can transfer the invested effect to another creature.